adam-core/src/test/scala/org/bdgenomics/adam/converters/AlignmentRecordConverterSuite.scala

```diff
@@ -222,7 +222,7 @@ class AlignmentRecordConverterSuite extends FunSuite {
 
   }
 
-  test("converting to fastq with unmapped reads") {
+  test("converting to fastq with unmapped reads when read reverse strand flag was set") {
```

Comment:
I would agree with @heuermh; specifically, I would fix the current test, and add a new test that tests the "unmapped + reverse strand flag" case.